lib/screens/friends/circle_friends_page/circle_friends_page.dart

```diff
@@ -35,31 +35,37 @@ class _CircleOfFriendPageState extends State<CircleOfFriendPage> {
   List<String> orderBy = [];
   String orderBySelected = '';
 
+  @override
+  void initState() {
+    init();
```

Comment:
no caso do initState o super.initState tem que estar no início do método

Reply:
Corrigido